### DISCUSSION TITLE
Account is not being verified

### DISCUSSION THREAD

**Discussion by RagaHarshitha** [community/community#50331] [GitHub Education]
Hi, this is Raga harshitha. I am trying to sign up the github education pack, I had added my Profile photo, bio but still it is not being verified.

**Comment by bbkx226** [comment]
Be patient and allow some time for the verification team to process your request. It may take a few days for them to review your application.

**Reply by RagaHarshitha** [reply]
I got a mail that they are unabale to verify the account.
May I know what are the things that our profile should contain.
![screenshot](https://user-images.githubusercontent.com/93828109/225975876-9c11449e-916c-4661-9baa-058f76d69114.png)

**Reply by bbkx226** [reply]
Hmm if that is the case, maybe you can try to reach out GitHub Support for the solutions

**Reply by RagaHarshitha** [reply]
okay thank you

**Comment by Anxiiie** [comment]
Same problem. Can't get verify too. I wrote a whole essay to describe everything but still nothing.
